test/e2e/framework/test_context.go

```diff
@@ -58,6 +59,11 @@ func validateFlags(t *TestContextType) {
 
 func ParseFlags() {
 	registerFlags(TestContext)
+	// Since Go 1.13, testing.Init() is required to register testing flags before calling flag.Parse()
+	var _ = func() bool {
+		testing.Init()
+		return true
+	}()
```

Comment:
but who's calling this _ var? 

any source explaining this?

Reply:
@mangelajo thanks for adding the pipefail #335, which solved the false-positive of passed travis builds. 

Regarding this PR, I will abandon it for now, since `testing.Init()` is not available on Go < 1.13 (our Travis configuration is using GO 1.11).

However, once we decide to move to GO 1.13, this fix will be required, otherwise go tests will fail on errors such as:

> flag provided but not defined: -test.v
> flag provided but not defined: -test.timeout
    
See more details in this Golang thread:
https://github.com/golang/go/issues/31859#issuecomment-489889428